### PR TITLE
Fix exp rename typo

### PR DESCRIPTION
**Pull request by mattseddon**
> You may disregard these recommendations if you used the **Edit on GitHub** button from dvc.org to improve a doc in place.

❗ Please read the guidelines in the [Contributing to the Documentation](https://dvc.org/doc/user-guide/contributing/docs) list if you make any substantial changes to the documentation or JS engine.

🐛 Please make sure to mention `Fix #issue` (if applicable) in the description of the PR. This causes GitHub to close it automatically when the PR is merged.

Please choose to [allow us](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to edit your branch when creating the PR.

Thank you for the contribution - we'll try to review it as soon as possible. 🙏

---

Noticed this whilst looking at https://github.com/iterative/vscode-dvc/pull/4616

### CODE REVIEW

File: content/docs/command-reference/exp/rename.md [hunk 1]
```diff
@@ -11,8 +11,8 @@ usage: dvc exp rename [-h] [-q | -v]
                       [experiment] [name]
 
 positional arguments:
-   experiment    Experiments to rename.
-   name          Name of new experiment.
+   experiment    Experiment to rename
+   name          New name for the experiment
```
Comment (dberenbaum) [inline content/docs/command-reference/exp/rename.md:15]:
```suggestion
   name          Name of new experiment
```

To match the CLI help text

File: content/docs/command-reference/exp/rename.md [hunk 2]
```diff
@@ -11,8 +11,8 @@ usage: dvc exp rename [-h] [-q | -v]
                       [experiment] [name]
 
 positional arguments:
-   experiment    Experiments to rename.
-   name          Name of new experiment.
+   experiment    Experiment to rename
+   name          Name of new experiment
```
Comment (dberenbaum) [inline content/docs/command-reference/exp/rename.md:15]: Sorry, just committed a suggestion to revert the change on this line, but was there a reason behind wanting to change it @mattseddon?

Comment (mattseddon) [reply content/docs/command-reference/exp/rename.md:15]: the experiment is still the same, it just has a new name. It is not a new experiment

Comment (dberenbaum) [inline content/docs/command-reference/exp/rename.md:15]:
```suggestion
   name          New name for the experiment
```

**Review by dberenbaum — APPROVED** [review]

**Comment by dberenbaum** [comment]
@mattseddon We should also change the CLI help text, although it's not critical.

**Comment by mattseddon** [comment]
> @mattseddon We should also change the CLI help text, although it's not critical.

I'll raise a PR
